### ISSUE TITLE
Does this support tree-shake?

### ISSUE THREAD

**Issue by pandafeeeder** [TechStark/opencv-js#41]

**Comment by ttt43ttt** [comment]
Isn't tree-shaking a function provided by the bundler tool (e.g. webpack)?

**Comment by neodescis** [comment]
> Isn't tree-shaking a function provided by the bundler tool (e.g. webpack)?

Yes and no. The bundler does the shaking, but a library must be shakable. There's quite a bit that goes into that! However, given that the bulk of this library is a web assembly module, I'm guessing there's not much that can be done.
